ros2bag/ros2bag/verb/record.py

```diff
@@ -60,6 +60,12 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
                   'Default it is zero, recording written in single bagfile and splitting '
                   'is disabled.'
         )
+        parser.add_argment(
```

Comment:
```suggestion
        parser.add_argument(
```

Reply:
Wow, I can't believe I missed that... Thanks for the catch!